apps/dashboard_web/lib/dashboard_web/live/components/headcounts.ex

```diff
@@ -24,8 +24,9 @@ defmodule DashboardWeb.Components.Headcounts do
       |> Map.put(:title, "Headcounts")
       |> Map.put(:product, :checkins)
       |> Map.put(:grid_width, 1)
-      |> Map.put(:data_key, :headcounts)
       |> Map.put(:data_number, assigns.total_count)
+      |> Map.put(:timeframe, "This week")
+      |> Map.put(:comparison_number, 800)
```

Comment:
Same as above.

Reply:
It is just me transferring your `arbitrary_comparison_number` from here: https://github.com/geolessel/pco-dashboard/pull/28/files#diff-23618e65c70c9ee74c567191884882f3L17